### DISCUSSION TITLE
Meet issues when generate init data.

### DISCUSSION THREAD

**Discussion by tianshuliu86** [deepmodeling/dpgen#1607] [Q&A]
Dear Developers,

When we tried to generate init data using following command:

$ dpgen init_bulk param.json machine.json

we got error alert as: "Name or service not known" 

![error](https://github.com/user-attachments/assets/719c8222-d424-4b00-9be3-698c27f853fc)

The system includes eight CF2Cl2 molecules and 40 atoms in total. 

Look forward to your response.

**Comment by njzjz** [comment]
Please check if your hostname is correct.
